test_server.py

```diff
@@ -42,11 +44,12 @@ def test_start_server(server_process):
 
 
 def test_response_ok():
-    assert server.response_ok() == STATUS200
+    assert b"200 OK" in server.response_ok().split(CRLF)[0]
 
 
 def test_response_error():
-    assert server.response_error() == STATUS500
+    assert b"500 Internal Server Error" in server.response_error(
+                                                        ).split(CRLF)[0]
```

Comment:
This is a confusing way to split the line. I would suggest either not splitting it, or trying something like this:

```
assert (b"500 Internal Server Error" in
        server.response_error().split(CRLF)[0])
```

You can encapsulate a statement in parentheses and then python will allow multiline splits.